frontend/public/components/catalog/catalog-page.tsx

```diff
@@ -210,7 +210,18 @@ export class CatalogListPage extends React.Component<CatalogListPageProps, Catal
           const chartName = chart.name;
           const tileName = `${_.startCase(chartName)} v${chart.version}`;
           const tileImgUrl = chart.icon || getImageForIconClass('icon-helm');
-          const chartURL = encodeURIComponent(_.get(chart, 'urls.0'));
+          const chartURL = _.get(chart, 'urls.0');
+          const encodedChartURL = encodeURIComponent(chartURL);
+          const markdownDescription = async () => {
+            let chartData;
+            try {
+              chartData = await coFetchJSON(`/api/helm/chart?url=${chartURL}`);
+            } catch (err) {
+              return '';
```

Comment:
`err` is unused.
Also probably better to return `null` `undefined` here.

Reply:
done.